src/plugins/teneo-api.js

```diff
@@ -172,7 +172,16 @@ export default function teneoApiPlugin(teneoApiUrl) {
 
             }).catch(
                 (error) => {
-                    EventBus.$emit(events.START_SPINNER);
+                    if (error instanceof SyntaxError) {
+                        console.error("Invalid JSON format. Please check your solution output.");
+                        EventBus.$emit(events.ADD_MESSAGE, {
+                            'type': 'system',
+                            'data': {
+                                'text': tmpVue.$t('message.unable_to_open_hyperlink')
```

Comment:
This could be any JSON error, couldn't it - not always hyperlink related?
To be sure that it is hyperlink related it would need to go inside the parseTeneoResponse: https://github.com/artificialsolutions/teneo-web-chat/blob/master/src/utils/parse-teneo-response.js#L103

Reply:
discussed externally with @JP-Artsol - this PR is obsoleted by TWCD-25

Reply:
I will close this PR, since the change was handled in TWCD-25
https://github.com/artificialsolutions/teneo-web-chat/blob/TWCD-25/src/plugins/teneo-api.js#L177